packages/smart-wallet/test/start-game1-proposal.js

```diff
@@ -0,0 +1,124 @@
+// @ts-check
+import { E } from '@endo/far';
+import { makeMarshal } from '@endo/marshal';
+
+console.warn('start-game1-proposal.js module evaluating');
+
+const { Fail } = assert;
+
+// vstorage paths under published.*
+const BOARD_AUX = 'boardAux';
+
+const marshalData = makeMarshal(_val => Fail`data only`);
+
+// avoid importing all of ERTP
```

Comment:
above it says a deep import would accomplish this (importing without all of it). yes or no?

Reply:
yes.

I think this dates from before I used `writeCoreProposal`, i.e. when `import` wouldn't work.